pyhoma/client.py

```diff
@@ -126,6 +126,19 @@ async def get_gateways(self, refresh: bool = False) -> List[Gateway]:
 
         return gateways
 
+    @backoff.on_exception(
+        backoff.expo, NotAuthenticatedException, max_tries=2, on_backoff=relogin
+    )
+    async def get_device_definition(self, deviceurl: str) -> JSON:
+        """
+        Retrieve a particular setup device definition
+        """
+        response = await self.__get(
+            f"setup/devices/{urllib.parse.quote_plus(deviceurl)}"
+        )
+
+        return response["definition"]
```

Comment:
Is definition always present? And what if we try to pull the definition for an unknown device or Hue/Zwave device? Are those scenarios tested and handled.

Reply:
- don't know
- don't know
- no

Reply:
I can change to `response.get("definition")`, that should keep us safe from "definition" not being present.

Reply:
fixed in https://github.com/iMicknl/python-tahoma-api/pull/51/commits/a5f12e5b4db6c0863d11c164778024f1071d2386